nomination/templates/nomination/url_add.html

```diff
@@ -238,6 +238,7 @@ <h4 class="panel-title">Add a Site/URL</h4>
                 initForm();
                 focusAddForm();
                 bindPreviewURL();
+							  new SlimSelect({select: '.multi-select'});
```

Comment:
tabs?

Reply:
ah! I missed these lines in the two templates. Yeah vim is inserting a single tab character at the beginning of the line and then I'm adding spaces. Not on purpose, I didn't realize it was doing that. I'll fix it.